src/test/java/build/buildfarm/worker/persistent/ProtoCoordinatorTest.java

```diff
@@ -126,6 +126,7 @@ public void testProtoCoordinator() throws Exception {
 
     pc.moveOutputsToOperationRoot(ctx, workRoot);
 
-    WorkerTestUtils.assertFilesExistExactly(opRoot, expectedOpRootFiles);
```

Comment:
This guy definitely doesn't belong here

Reply:
Moved to https://github.com/bazelbuild/bazel-buildfarm/pull/1785